site/examples/markdown-preview.tsx

```diff
@@ -70,38 +70,38 @@ const Leaf = ({ attributes, children, leaf }) => {
         font-style: ${leaf.italic && 'italic'};
         text-decoration: ${leaf.underlined && 'underline'};
         ${leaf.title &&
-          css`
-            display: inline-block;
-            font-weight: bold;
-            font-size: 20px;
-            margin: 20px 0 10px 0;
-          `}
+        css`
+          display: inline-block;
+          font-weight: bold;
+          font-size: 20px;
+          margin: 20px 0 10px 0;
+        `}
         ${leaf.list &&
-          css`
-            padding-left: 10px;
-            font-size: 20px;
-            line-height: 10px;
-          `}
+        css`
```

Comment:
This indentation is a little odd, is this all new-prettier's doing? One would normally think the second operand of a conjunction would be indented relative to the previous if it must be on a new line...
What if you try to put `css` immediately after the `&&` like so?
```
${leaf.title && css`
```
Does that change the indentation a bit? I

Reply:
All code changes are from running `yarn lint --fix`. I can try manually changing the line manually and re-running, but will be a bit before I update.